.github/workflows/trash-cleanup-release.yml

```diff
@@ -0,0 +1,66 @@
+name: Trash Cleanup Release
```

Comment:
I guess we don't care now. In the future we may want to have a common GHA that will cover release of every of our extensions, I presume.